packager/media/codecs/ec3_audio_util.h

```diff
@@ -27,6 +27,19 @@ bool CalculateEC3ChannelMap(const std::vector<uint8_t>& ec3_data,
 ///         success; otherwise 0 is returned.
 size_t GetEc3NumChannels(const std::vector<uint8_t>& ec3_data);
 
+/// Parse data from EC3Specific box, calculate EC3 channel map and then
+/// obtain channel configuration descriptor value with MPEG scheme based on
+/// ETSI TS 102 366 V1.4.1 Digital Audio Compression (AC-3, Enhanced AC-3)
+/// Standard, Table I.1.1.
+bool CalculateEC3ChannelMPEGValue(const std::vector<uint8_t>& ec3_data,
+                                  uint32_t& ec3_channel_mpeg_value);
```

Comment:
Can you return the output in pointer for consistency, i.e.

bool CalculateEC3ChannelMPEGValue(const std::vector<uint8_t>& ec3_data,
                                  uint32_t* ec3_channel_mpeg_value);

Similar below.

Reply:
Sure. I will update CalculateEC3ChannelMPEGValue and GetEc3JocInfo.